File: tests/kits/c/rock.c

#include "kit.h"
#include <stdio.h>

int main() {
  // initialize
  agent_initialize();
  // we have agent of type Agent * available now
  while(1) {
    // play rock
    fprintf(stdout, "R\n");
    // end turn
    agent_end_turn();
    // wait for updates
    agent_update();
  }
  
  return 0;
}

File: tests/kits/js/normal/rockdir/rock.js
const kit = require('./kit');
const agent = new kit.Agent();
agent.initialize().then(async () => {
  while(true) {
    console.error("error R");
    console.log('R'); // tell the match you want to play Rock in the game
    agent.endTurn(); // end turn
    await agent.update(); // wait for updates
  }
});